docs/support-bundle-guide.md

```diff
@@ -0,0 +1,253 @@
+      name: support-bundle-secret
+      namespace: default # antrea-controller must be given the permission to read Secrets in "default" Namespace.
+EOF
+```
+
+For more information about the supported fields in a "SupportBundleCollection"
+CR, please refer to the [CRD definition](/build/charts/antrea/crds/supportbundlecollection.yaml)
+
+You can check the status of `SupportBundleCollection` by running
+`kubectl get supportbundlecollections $Name-of-SupportBundleCollection -ojson`.
+The following example shows a successful realization of `SupportBundleCollection`.
+`desiredNodes` shows the number of Node/ExternalNode is expected in this request,
+while `collectedNodes` shows the number of Node/ExternalNode which has already
+uploaded bundle files to the target file server. `collectedNodes` and `desiredNodes`
+should match the number of Node/ExternalNode you want to collect support bundle
+if the collection is succeeded.
+
+If the following two conditions are presented, it means a bundle collection is
+succeeded,
+
+1. "Completed" is true
+2. "CollectionFailure" is false.
+
+If any expected Node/ExternalNode failed to upload the bundle files in the
+required time, the "CollectionFailure" condition will be set to true.
+
+```bash
+$ kubectl get supportbundlecollections support-bundle-for-nodes -ojson
+
+...
+ "status": {
+        "collectedNodes": 1,
+        "conditions": [
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "True",
+                "type": "Started"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "BundleCollected"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "False",
+                "type": "CollectionFailure"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "Completed"
+            }
+        ],
+        "desiredNodes": 1
+    }
+```
+
+The collection should include three tarballs. To access these files, you can
+download the files from the SFTP server `yourtestdomain.com`. There will be two
+tarballs for `support-bundle-for-nodes` including
+"support-bundle-for-nodes_worker1.tar.gz" and "support-bundle-for-nodes_worker2.tar.gz",
+and two for `support-bundle-for-vms` including "support-bundle-for-vms_vm1.tar.gz"
+and "support-bundle-for-vms_vm2.tar.gz" in the `/root/test` folder. Run the
+`tar xvf $TARBALL_NAME` command to extract the files from the tarballs.
+
+## List of collected items
+
+Depending on the methods you use to collect the support bundle, the contents in
+the bundle may differ. The following table shows the differences.
+
+We use `agent`,`controller` and `outside` to represent running command
+`antctl supportbundle` in Antrea Agent, Antrea Controller and out-of-cluster
```

Comment:
I feel changing "and" to "," reads better.

Reply:
updated.